### PR TITLE
New seed

### PR DESCRIPTION
**Pull request by theofficetroll**
New seed and generator scripts. Built off of existing, making it more robust.

https://trello.com/c/EGyOOMUB

### CODE REVIEW

File: database/helpers/seed.js [hunk 1]
```diff
@@ -4,28 +4,29 @@ var generator = require('./generator.js');
...
+          styleId: (i === 0 ? limit : i) * 3 + 1,
+          price: generator.getPrice(),
+          options: generator.getOptions()
 
         },
         {
-          styleId: '003',
-          price: generator.getPrice(i),
-          options: generator.getOptions(i)
+          styleId: styleId: (i === 0 ? limit : i) * 3 + 2,
```
Comment (TandyN) [inline database/helpers/seed.js]: Syntax error?

**Review by TandyN — CHANGES_REQUESTED** [review]
Notes:
I see that you've made some adjustments to the generating script.

Change Request:
- Please address  comment.

Question:
- The seeding script is using mongoose right now. I'm assuming you plan on making changes to that to match your chose databases later?

File: database/helpers/seed.js (hunk 1 above)
Comment (theofficetroll) [reply database/helpers/seed.js]: Verily! Thank you!

**Comment by theofficetroll** [comment]
Per Leslie, first need to update the generator/ seed scripting to work with current database.
Then set up new databases and variant seeds to work with each of those databases.

**Pull request merged**